lib/piv/views/slot_dialog.dart

```diff
@@ -40,6 +61,32 @@ class SlotDialog extends ConsumerWidget {
       return const FsDialog(child: CircularProgressIndicator());
     }
 
+    TableRow detailRow(String title, String value) {
```

Comment:
If possible rewrite this as a StatelessWidget subclass. Using functions which return Widget is an anti-pattern in flutter.

Reply:
Nope. We need a list of TableRow instances.

Reply:
I see now 👍🏻